src/JoomlaBrowser.php

```diff
@@ -855,4 +856,16 @@ public function verifyAvailableTabs($expectedTabs, $tabsLocator = ['xpath' => "/
 		$I->assertEquals($expectedTabs, $actualArrayOfTabs, "Tab Labels do not match on edit view of" . $url);
 		$I->debug('Verify the Tabs');
 	}
+
+    /**
+     * Hide the statistics info message
+     *
+     * @note: doAdminLogin() before
+     */
+    public function disablestatistics()
+    {
+        $I = $this;
+        $this->debug('I click on never');
+        $I->click(['link' => 'Never']);
```

Comment:
before clicking we need to make sure the link is already available.

we need a:

`$I->waitForElement(['link' => 'Never'], 60)`